workflow/sample_files/base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-autosize.xml

```diff
@@ -51,6 +51,12 @@
           <ConditionedFloorArea>2700.0</ConditionedFloorArea>
           <ConditionedBuildingVolume>21600.0</ConditionedBuildingVolume>
         </BuildingConstruction>
+        <extension>
+          <AdditionalProperties>
+            <HeatPump>YORK HMH7</HeatPump>
+            <HeatPumpAHRICert>208613158</HeatPumpAHRICert>
+          </AdditionalProperties>
+        </extension>
```

Comment:
Added Names and AHRI#s so we can more easily look up these products in the NEEP database in the future.